src/js/pages/services/ServicesTab.js

```diff
@@ -55,6 +56,18 @@ var ServicesTab = React.createClass({
     this.store_listeners = [{name: 'dcos', events: ['change']}];
   },
 
+  componentDidMount: function () {
+    let {state} = this;
+    Object.keys(DEFAULT_FILTER_OPTIONS).forEach((saveStateKey) => {
+      const queryParams = this.getQueryParamObject();
+      if (queryParams != null &&
+        queryParams[saveStateKey] &&
```

Comment:
This test introduces a bug; it will only update if the saveStateKey is already in the queryParams.